server/app.js

```diff
@@ -4,9 +4,6 @@ var bodyParser = require("body-parser");
 
 var exampleRoutes = require("./routes/example");
 var calendarRoutes = require("./routes/calendar");
-var employeesRoutes = require("./routes/employees");
```

Comment:
This looks wrong to me. I think you were trying to resolve some merge conflicts but we don't want to delete these routes! You can just copy and paste the code github is saying was deleted back into the file.